package/network/services/dnsmasq/files/dnsmasq.init

```diff
@@ -362,7 +363,20 @@ dhcp_add() {
 		DNS_SERVERS="$DNS_SERVERS $dnsserver"
 	}
 
-	append_bool "$cfg" ignore "--no-dhcp-interface=$ifname" && return 0
+	append_bool "$cfg" ignore "--no-dhcp-interface=$ifname" && {
+		if [ $ADD_LOCAL_GATENAME -gt 0 ] ; then
+			local dashedname="$( echo $ifname | sed 's/\./-/g' )"
```

Comment:
I get frustrated with so many (non std) shell implementations, that I take the long way first to guarantee function. DONE